packages/react-scripts/package.json

```diff
@@ -48,6 +48,7 @@
     "ts-loader": "^2.3.7",
     "tsconfig-paths-webpack-plugin": "^2.0.0",
     "tslint": "^5.7.0",
+    "tslint-config-prettier": "^1.10.0",
```

Comment:
I don't think we should add this. We should aim to be as flexible as possible.

Reply:
It seemed you agreed to using this package in #77, that's why I added it. If I understand you correctly you now want to not use Prettier but instead use just the `tslint:recommended` and `tslint-react` presets?

Reply:
Apologies, yes no style rules from prettier.